### PR TITLE
added distanceToReferencePlane-function

### PR DESCRIPTION
**Pull request by wsteenhu**
Hi Sudarshan,
I think I managed to at least set a skeleton for the implementation of the distanceToReferencePlane-function.
I tried to align with your conventions (so copy-pasted quite a lot :)).
I verified that, given a matrix with x, y, z coordinates for a group of samples (created using an ordination like PCoA and selecting the first 3 columns), I can generate the same results using our R implementation compared to the original Python-code (I can share the benchmark results if you like).
One open question is whether or not we should check if the order of sample ids in the phyloseq is the same as in the distance matrix (I added a small line to check, may also apply to some of the other functions that take both phyloseq and distance matrix as input).
Looking forward to your comments/edits!

### CODE REVIEW

File: R/distanceToReferencePlane.R [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+#' dist.mat <- phyloseq::distance(WirbelJ_2018, "bray")
+#' ref.samples <- sample_names(subset_samples(WirbelJ_2018,
+#'                                            disease == "healthy"))
+#' dtrp.results <- distanceToReferencePlane(WirbelJ_2018,
+#'                                          dist_mat = dist.mat,
+#'                                          reference_samples = ref.samples)
+#'
+#' head(dtrp.results)
+#'
+#' @references
```
Comment (microsud) [inline R/distanceToReferencePlane.R:39]: Also, include the reference to the python code GitHub repository

**Comment by microsud** [comment]
I have done the review. Looks almost done with minor comments. You can see all my comments the tab `Files Changed` Make sure you update your branch with this synced version before making any changes. 
Edite: Forgot to mention that you also need to add this function in the https://github.com/microsud/dysbiosisR/blob/master/R/dysbiosisOverview.R function. This provides a quick list of supported measures by the `dysbiosisR` package. 

Cheers,
Sudarshan

**Comment by wsteenhu** [comment]
Hi Sudarshan, thanks for this. I resolved the issue above and added the function to the overview. I however cannot see any comments under the `Files changed`-tab (this seems to only show the diff of my previous commit). Anything I am missing or can I push again?

**Comment by microsud** [comment]
Thanks! Wierd I se it here https://github.com/microsud/dysbiosisR/pull/2/files#diff-9246663636b0efdfb05df0459c0404c67d5f90652860ffa783f0c80da3b93552

if you cannot find then push your commit to this branch and I will check it. Thanks again for working on this.

**Comment by wsteenhu** [comment]
Hi, yeah odd, I do see you went through the files (see some blue bars), but can't find any comments... Sorry about that.
I now added the function to the overview and added a reference to the python-implementation in the description as you suggested. Let me know if it needs more work!

**Comment by microsud** [comment]
Great and thanks for this contribution!

**Pull request merged**